Automattic/pocket-casts-ios
# Refactor IAP related code

Project #1373

This PR refactor the IAP related code to use enum better (avoid rawValues uses) and group related types on a project group.
It also adds a unit test to validate current code

## To test

Steps to test your PR.

- Check if the flows to Purchase a subscription show the correct values for payments
- Go to Profile
- Tap Account
- Check if values show for Plus and Patron are correct
- Tap Subscribe to Plus
- Check if values for Plus monthly and yearly are correct
- Swipe to Patron
- Check if values for Patron monthly and yearly are correct
- Tap on Subscribe and check if IAP flow starts correctly

## Checklist
